tests/single/gas/test_pre_verification_gas_calculation.py

```diff
@@ -75,7 +75,7 @@ def test_simple_paymaster(w3, entrypoint_contract):
 @pytest.mark.usefixtures("manual_bundling_mode")
 @pytest.mark.parametrize("dynamic_length_field_name", dynamic_length_field_names)
 @pytest.mark.parametrize("field_length", field_lengths)
-def test_pre_verification_gas_calculation(
+def xtest_pre_verification_gas_calculation(
```

Comment:
better"
@pytest.mark.skipif(os.getenv("SLOW") == None, reason="very slow. run manually with SLOW=1")

Reply:
It's temporary, not meant to stay that way